### PR TITLE
Feat: [DS-288] Added support for action link & closable horizontal ap…

### PR DESCRIPTION
**Pull request by vicky-comeau**
…pearance

### CODE REVIEW

File: packages/Avatar/src/__snapshots__/Avatar.test.tsx.snap [hunk 1]
```diff
@@ -4,6 +4,7 @@ exports[`Avatar It should render a snapshot 1`] = `
 <DocumentFragment>
   <div
     class="ids-avatar ids-avatar--medium"
+    data-private="true"
```
Comment (fraincs) [inline packages/Avatar/src/__snapshots__/Avatar.test.tsx.snap:7]: is that from another PR?

File: packages/Dropdown/src/__snapshots__/Dropdown.test.tsx.snap [hunk 2]
```diff
@@ -2,7 +2,9 @@
 
 exports[`Dropdown It should render without error and a snapshot 1`] = `
 <body>
-  <div>
+  <div
+    data-floating-ui-inert=""
```
Comment (fraincs) [inline packages/Dropdown/src/__snapshots__/Dropdown.test.tsx.snap:6]: are the empty quotes necessary?

File: packages/Avatar/src/__snapshots__/Avatar.test.tsx.snap (hunk 1 above)
Comment (vicky-comeau) [reply packages/Avatar/src/__snapshots__/Avatar.test.tsx.snap:7]: I didn't refresh the snapshot last time. I don't remember it complaining.. weird. so you can ignore this.

File: packages/Dropdown/src/__snapshots__/Dropdown.test.tsx.snap (hunk 2 above)
Comment (vicky-comeau) [reply packages/Dropdown/src/__snapshots__/Dropdown.test.tsx.snap:6]: it's a snapshot. auto generated

**Review by fraincs — APPROVED** [review]

**Pull request merged**